test/integration/targets/zypper/tasks/zypper.yml

```diff
@@ -1,3 +1,25 @@
+# https://github.com/ansible/ansible/issues/27988
+- name: download apache-rpm-macros-control
+  shell: zypper --non-interactive install -d apache-rpm-macros-control
+- name: move rpm package  to /tmp and rename to /tmp/apache-rpm-macros-control.rpm
```

Comment:
Don't use `/tmp`, but the `{{output_dir | expanduser}}` to not write files outside the test environment.

Reply:
There is already a valid rpm built inside the test cases. It would be easier to use that file instead of downloading/moving an external file.